mainwindow.ui

```diff
@@ -35,11 +110,7 @@
     - close search field with `Escape`
 - and much more...
 
-```cpp
-#include &lt;iostream&gt;
-int main () {
-    std::cout &lt;&lt; &quot;hello&quot;
-}
+```css
```

Comment:
Did you want to remove the code and leave an empty block?

Reply:
I left it in for now so that you can test it easily.

Reply:
Ah, ok